include/oneapi/dpl/pstl/experimental/internal/for_loop_impl.h

```diff
@@ -161,27 +161,27 @@ class __reduction_pack
     }
 };
 
-// Sequenced version of for_loop_n
+// Sequenced + vectorized version of for_loop_n
 template <class _Tag, typename _ExecutionPolicy, typename _Ip, typename _Size, typename _Function, typename... _Rest>
 void
 __pattern_for_loop_n(_Tag, _ExecutionPolicy&&, _Ip __first, _Size __n, _Function __f, __single_stride_type,
                      _Rest&&... __rest) noexcept
 {
-    static_assert(__is_backend_tag_serial_v<_Tag> || __is_backend_tag_parallel_forward_v<_Tag>);
+    static_assert(__is_backend_tag_v<_Tag>);
```

Comment:
It seems that should be `static_assert(__is_backend_tag_serial_v<_Tag> || __is_backend_tag_parallel_forward_v<_Tag>);`

Reply:
Fixed